rotkehlchen/tests/unit/test_globaldb.py

```diff
@@ -761,6 +767,71 @@ def test_global_db_reset(globaldb):
     )
     GlobalDBHandler().edit_evm_token(one_inch_update)
 
+    # Add some data to the tables that reference assets to make sure that it is not
+    # touched by the reset
+    historical_price = HistoricalPrice(
+        from_asset=A_ETH,
+        to_asset=A_DAI,
+        source=HistoricalPriceOracle.COINGECKO,
+        timestamp=1337,
+        price=ONE,
+    )
+    GlobalDBHandler().add_single_historical_price(historical_price)
+    GlobalDBHandler().add_user_owned_assets([A_ETH, A_DAI, A_CRV])
+    db_custom_assets = DBCustomAssets(database)
+    custom_asset = CustomAsset.initialize(
+        identifier=str(uuid4()),
+        name='My favorite lamborgini',
+        custom_asset_type='Sport car',
+        notes='It is so fast and so furious!',
+    )
+    db_custom_assets.add_custom_asset(custom_asset)
```

Comment:
> @LefterisJP if during your big refactor you also need to change packaged globaldb, then mentioning my changes so that we can merge them easier:
> Create assets_collection with id 1, name DAI and symbol DAI
> Put in this collection's mapping mainnet DAI and optimism DAI

Please undo this packaged.d change. I don't want any changes in the packaged global.db for this PR. It makes no sense to add them for this test. I thought such collections did already exist.

Perhaps the test can just check they are removed? Since they are not in the packaged DB? So the test can add them and then it sees they are removed?

And if/when we add any collections we need to modify this test